### PR TITLE
Update BLockListEditor

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Add ModelsBuilder Examples

### CODE REVIEW

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Block-List-Editor/index.md [hunk 1]
```diff
@@ -148,6 +156,15 @@ Example:
 
 ```
 
+With ModelsBuilder:
+
+```csharp
+
+// Output the value of field with alias 'heading' from the Element Type selected as Content section
+<h1>@content.Heading)</h1>
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Block-List-Editor/index.md:164]:
```suggestion
<h1>@content.Heading</h1>
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
Looks good @erikjanwestendorp 💪 

I've made a small suggestion - looks like there was one parenthesis too many.

**Comment by erikjanwestendorp** [comment]
@sofietoft Oops.. Just committed your suggestion 👍

**Review by sofietoft — APPROVED** [review]
Great! 😄 

I'll merge it right in.
Thanks again for updating the article @erikjanwestendorp !

**Pull request merged**